bsp/Infineon/libraries/HAL_Drivers/drv_rtc.c

```diff
@@ -94,6 +94,11 @@ static rt_err_t ifx_rtc_get_timeval(struct timeval *tv)
 
 static rt_err_t _rtc_init(void)
 {
+#ifdef   BSP_RTC_USING_LSE
```

Comment:
#ifdef BSP_RTC_USING_LSE
```suggestion
#ifdef BSP_RTC_USING_LSE
```